pxr/usdImaging/usdviewq/appController.py

```diff
@@ -41,7 +41,8 @@
 from pxr.UsdUtils.constantsGroup import ConstantsGroup
 
 # UI Components
-from ._usdviewq import Utils
+with Tf.WindowsImportWrapper():
```

Comment:
I thought it'd be nice to avoid having to use the import wrapper outside of Tf so I added commit d9580dbec27 to make sure we initialize usdviewq with Tf.PreparePythonModule. I tested this out and it seems to work, so I think the changes in usdviewq can be reverted.